src/main/java/me/rigamortis/seppuku/impl/module/combat/NoCrystalModule.java

```diff
@@ -133,37 +124,53 @@ private int findStackHotbar(Block type) {
             final ItemStack stack = Minecraft.getMinecraft().player.inventory.getStackInSlot(i);
             if (stack.getItem() instanceof ItemBlock) {
                 final ItemBlock block = (ItemBlock) stack.getItem();
```

Comment:
The above 3 lines of code is used nearly identically 3 times in this file.  Why not make one function that just compares an Item to the Block type?